src/bidiMapper/domains/events/EventManager.ts

```diff
@@ -275,24 +285,13 @@ export class EventManager {
     }
     return result.sort((e1, e2) => e1.id - e2.id);
   }
-}
 
-const EVENT_NAMES = new Set([
-  // keep-sorted start
-  ...Object.values(ChromiumBidi.BiDiModule),
-  ...Object.values(ChromiumBidi.BrowsingContext.EventNames),
-  ...Object.values(ChromiumBidi.Log.EventNames),
-  ...Object.values(ChromiumBidi.Network.EventNames),
-  ...Object.values(ChromiumBidi.Script.EventNames),
-  // keep-sorted end
-]);
-
-function checkEventName(name: string) {
-  if (
-    !EVENT_NAMES.has(name as never) &&
-    !name.startsWith('cdp.') &&
-    name !== 'cdp'
-  ) {
-    throw new InvalidArgumentException(`Unknown event: ${name}`);
+  static isSupportedEvent(name: string) {
+    if (
+      !EVENT_NAMES.has(name as never) &&
+      !name.startsWith(ChromiumBidi.BiDiModule.Cdp)
```

Comment:
Should we re-add the `.` suffix to `startsWith`?

Although highly unlikely, this would match "cdpee". If we re-add the dot, then `cdp.` ensures uniqueness.

Reply:
Alternatively: name + split on "." + get first element + check that it equals Cdp

The first approach is simpler though

Reply:
We need to match `cdp` as well as `cdp` what is why I did it as so, but you are correct we should match for both individually.